snapshot/src/test/java/io/zeebe/snapshots/impl/FileBasedReceivedSnapshotTest.java

```diff
@@ -109,8 +109,8 @@ public void shouldReceiveChunk() throws Exception {
     // then
     assertThat(receivedSnapshot.getPath())
         .as("the received snapshot directory contains the applied chunk")
-        .isDirectoryContaining(
-            p -> p.getFileName().toString().equals(expectedChunk.getChunkName()));
+        .isDirectoryNotContaining(
+            p -> !p.getFileName().toString().equals(expectedChunk.getChunkName()));
```

Comment:
Here we should use `isDirectoryContaining` because we want to check if the chunk exists. We don't want to check that only one chunk exists.